portal/src/util/date.ts

```diff
@@ -38,3 +38,7 @@ export function parseDate(isoDate: string): Date {
     zone: "UTC",
   }).toJSDate();
 }
+
+export function getCurrentDate(): Date {
+  return new Date(Date.now());
+}
```

Comment:
Remove this function and write `new Date()` to obtain the current timestamp represented in a Date object.